prow/pod-utils/downwardapi/jobspec.go

```diff
@@ -98,13 +98,14 @@ const (
 	buildIDEnv     = "BUILD_ID"
 	prowBuildIDEnv = "BUILD_NUMBER" // Deprecated, will be removed in the future.
 
-	repoOwnerEnv   = "REPO_OWNER"
-	repoNameEnv    = "REPO_NAME"
-	pullBaseRefEnv = "PULL_BASE_REF"
-	pullBaseShaEnv = "PULL_BASE_SHA"
-	pullRefsEnv    = "PULL_REFS"
-	pullNumberEnv  = "PULL_NUMBER"
-	pullPullShaEnv = "PULL_PULL_SHA"
+	repoOwnerEnv       = "REPO_OWNER"
+	repoNameEnv        = "REPO_NAME"
+	pullBaseRefEnv     = "PULL_BASE_REF"
+	pullBaseShaEnv     = "PULL_BASE_SHA"
+	pullRefsEnv        = "PULL_REFS"
+	pullNumberEnv      = "PULL_NUMBER"
+	pullPullShaEnv     = "PULL_PULL_SHA"
+	pullHeadRefNameEnv = "PULL_HEAD_REF_NAME"
```

Comment:
How about `PULL_HEAD_REF` for naming consistency?

Reply:
Sounds fine to me, I updated it.